src/Identity/Startup.cs

```diff
@@ -138,6 +140,10 @@ public void ConfigureServices(IServiceCollection services)
         services.AddDefaultServices(globalSettings);
         services.AddCoreLocalizationServices();
 
+        // TODO: Remove when OrganizationUser methods are moved out of OrganizationService, this noop dependency should
+        // TODO: no longer be required - see PM-1880
```

Comment:
PM-1880 won't fix this because `IServiceAccountRepository` will still be a project depedency, it'll just be moved to the command instead

Reply:
But the command will be called directly from the Api controller, so it'll only be registered in the Api project, so we won't need it or its dependencies here (in Identity). Or in the other random projects we've added it to here.